modules/recording/recording/libs/download.ts

```diff
@@ -0,0 +1,98 @@
+import { PDFDocument, rgb, StandardFonts } from "pdf-lib";
+import { saveAs } from "file-saver";
+
+class Download {
```

Comment:
Why to start from templates branch? If we had started from Main, that can be easly merged, small and safe

Reply:
I think perhaps we merge into templates once that's done then merge into main.

I'll take note of this for the future